crankshaft-docker/src/service/builder.rs

```diff
@@ -186,15 +190,18 @@ impl Builder {
             .await
             .map_err(Error::Docker)?;
 
+        let id = response.id.expect("service must have an identifier");
```

Comment:
I was curious why this is an optional, so I did some digging. It appears that, in the latest version of the Docker API specification, the returned ID is not optional ([source](https://docs.docker.com/reference/api/engine/version/v1.48/#tag/Service/operation/ServiceCreate)). I find it odd that `bollard` makes this field optional.

Reply:
Just a comment, you can immediately resolve upon reading.